content/en/getting_started/synthetics/api_test.md

```diff
@@ -55,7 +59,7 @@ The example below demonstrates how to create an [HTTP test][3], a subtype of [si
       In this example, no specific advanced option is needed.
     - You can set tags such as `env:prod` and `app:shopist` on your test. Tags allow you to keep your test suite organized and quickly find tests you're interested in on the homepage.
 
-5. Click **Test URL** to trigger a sample test run.
+6. Click **Test URL** to trigger a sample test run.
```

Comment:
I think we're missing a step to click **Create Test** here, regardless of method?